q2_quality_control/_threshold_graph/_visualizer.py

```diff
@@ -60,31 +95,112 @@ def decontam_score_viz(output_dir, decontam_scores: qiime2.Metadata,
     gray_lab_arr = []
     red_lab_arr = []
     blue_lab_arr = []
+    data_arr = []
```

Comment:
What are each of these?

Reply:
comments added and depreciated arrays were removed

addressed in [adfe12a](https://github.com/qiime2/q2-quality-control/pull/82/commits/adfe12ad4213c9923face4267028d18902427b8a)